requirements/constraints.txt

```diff
@@ -112,5 +112,5 @@ docutils<0.20
 # greater version has dropped few dependencies. Fix this in other ticket.
 drf-yasg<1.21.6
 
-# incremental upgrade approach. After successfull deploy remove this pin and try latest version.
-djangorestframework<3.14.0
+# Adding pin to avoid any major upgrade
+djangorestframework<3.15.0
```

Comment:
Hello. This is an old PR, but I noticed it recently.

Either the comment is off or the pin is off. If the comment is accurate, this should be `djangorestframework<4.0.0`, correct?